.circleci/config.yml

```diff
@@ -126,7 +126,24 @@ jobs:
       - run:
           name: Deploy docs to gh-pages branch
           command: gh-pages --message "[skip ci] updates" --dist /tmp/_html
-
+  looker-deploy:
+    executor: gcp-cli/google
+    steps:
+      - gcp-cli/setup:
+          use_oidc: true
+          workload_identity_pool_id: GCP_WIP_ID
+          workload_identity_pool_provider_id: GCP_WIP_PROVIDER_ID
+          google_project_number: GCP_PROJECT_ID
+          service_account_email: GCP_SERVICE_ACCOUNT_EMAIL
+      - run:
+          name: Trigger looker DAG in Airflow to deploy lookml
+          command: >
+            ID_TOKEN=$(gcloud auth print-identity-token --impersonate-service-account ${GCP_SERVICE_ACCOUNT_EMAIL}) &&
+            DAGRUN_NOTE="DAG triggered by **[${CIRCLE_USERNAME}](https://github.com/${CIRCLE_USERNAME})** from ${CIRCLE_PROJECT_USERNAME}/${CIRCLE_PROJECT_REPONAME} CI build [${CIRCLE_BUILD_NUM}](${CIRCLE_BUILD_URL})" &&
+            curl --location --request POST "https://us-west1-moz-fx-telemetry-airflow-prod.cloudfunctions.net/ci-external-trigger"
```

Comment:
todo: this cloud function needs to be updated to take the `dag_id` parameter. Or a separate cloud function needs to be created for this

Reply:
depends on https://github.com/mozilla-it/dataservices-infra/pull/282